tools/shared.py

```diff
@@ -472,6 +472,15 @@ def find_temp_directory():
 def get_emscripten_version(path):
   return open(path).read().strip().replace('"', '')
 
+# Returns true if Emscripten is running in 'strict' mode, in which deprecated compiler features are not supported.
+def is_emscripten_strict():
+  if os.environ.get('EMCC_STRICT') and int(os.environ.get('EMCC_STRICT')) != 0: return True
```

Comment:
throwing might be ok, but it'll throw something like `ValueError: invalid literal for int() with base 10` which is not very clear. and in all other places, we accept `'0'` as "no", and everything else, including `"a"` as yes.